draft-ietf-rtcweb-jsep.xml

```diff
@@ -1243,6 +1243,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           and/or pending remote description according to the rules defined for
           Trickle ICE. Connectivity checks will be sent to the new candidate.</t>
 
+          <t>This method can also be used to provide an end-of-candidates
+          indication to the ICE Agent, as defined in
+          <xref target="I-D.ietf-ice-trickle"/>.</t>
```

Comment:
@ekr, I think this is worth spelling out since it wasn't always this way, at least not in the w3c spec.

@taylor-b / @pthatcherg not sure that we should specify null, since that's a bit of a w3c-ism; rather, I think the generic notion that we can supply end-of-candidates here should be sufficient. This would match the advice given in https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-13#section-3.4.1, where it simply says that an event is generated to indicate end-of-candidates.